src/server/pps/server/worker_rc.go

```diff
@@ -612,6 +619,20 @@ func (a *apiServer) getWorkerOptions(ptr *pps.EtcdPipelineInfo, pipelineInfo *pp
 	}, nil
 }
 
+func (a *apiServer) setupWorkerPachctlAuth(ctx context.Context, ptr *pps.EtcdPipelineInfo, pipelineInfo *pps.PipelineInfo) error {
```

Comment:
I would include `secret` in this method name, just to make it explicit that it's installing a k8s secret. Like `createWorkerPachctlSecret`